launch/sonar_tank_blueview_p900_nps_multibeam.launch

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0"?>
 <launch>
   <include file="$(find gazebo_ros)/launch/empty_world.launch">
-    <arg name="world_name" value="$(find nps_uw_sensors_gazebo)/worlds/sonar_tank_blueview_p900.world"/>
+    <arg name="world_name" value="$(find nps_uw_sensors_gazebo)/worlds/sonar_tank_blueview_p900_nps_multibeam.world"/>
```

Comment:
Just a comment as I'm trying to get things set up and running. The verbose flag (as instructed in the wiki) to the launch file isn't doing anything right now.
I added it locally:
```
  <arg name="verbose" default="false"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find nps_uw_sensors_gazebo)/worlds/sonar_tank_blueview_p900_nps_multibeam.world"/>
    <arg name="verbose" value="$(arg verbose)"/>
  </include>
```